app/models/service_template.rb

```diff
@@ -43,7 +43,9 @@ class ServiceTemplate < ApplicationRecord
   include ArchivedMixin
   include CiFeatureMixin
   include_concern 'Filter'
+  include_concern 'Copy'
 
+  validates :name, :presence => true
```

Comment:
Do we need a migration to fix any existing records that don't have a name?

Reply:
That's an excellent question.

Reply:
I've heard naming is hard.  Good eyes @bdunne 👀

Reply:
There's a limit though, only one data migration per team person per year, my quota is filled.

Reply:
> There's a limit though, only one data migration per team person per year, my quota is filled.

But now you know the pattern :smile:

Reply:
@d-m-u and miss out on all the fun in writing the spec for the data migration?